content/_data/changelogs/lts.yml

```diff
@@ -1770,6 +1770,40 @@
       issue: 51955
       message: >
         Developer API: Prevent <code>NullPointerException</code> in <code>SlaveComputer#setChannel(Channel,OutputStream,Channel.Listener)</code> with <code>null</code> <code>OutputStream</code>.
+- version: "2.121.3"
+  date: 2018-08-15
+  changes:
+  - type: rfe
+    pull: 3522
+    issue: 52159
+    message: >
+      Robustness: Don't break queue processing when the configured queue sorter throws exceptions.
+  - type: rfe
+    pull: 3542
+    issue: 52534
+    message: >
+      Whitelist <code>java.time.Ser</code> for use in XStream (XML serialization) and Remoting (agent communication).
+  - type: bug
+    pull: 3354
+    issue: 46248
+    message: >
+      Fix a potential deadlock between queue maintenance and asynchronous execution.
+  - type: bug
+    pull: 3482
+    issue: 51777
+    message: >
+      Security hardening: Prevent files in <code>tar</code> archives from being written to a path outside the destination directory.
+  - type: bug
+    pull: 3496
+    issue: 28683
+    message: >
+      Dynamically loaded plugins now have any <code>PeriodicWork</code>/<code>AperiodicWork</code> extensions scheduled.
+  - type: rfe
+    pull: 3480
+    issue: 52417
+    message: >
+      Developer: Remove <code>hudson.FilePath#copyFromRemotely(URL)</code> Beta API.
```

Comment:
Do we really remove it? Maybe restrict actually? CC @jglick @olivergondza

Reply:
No, It [became restricted](https://github.com/jenkinsci/jenkins/commit/4cb86a5c3055d39cf836a7d662f53e316a6a1cd7) not to break binary compatibility on user side. It *will* break developers on 2.121.3 trying to build against the API but that is the point here. The method will not be there in further LTS releases but that should be no surprise, too.